server/manager/src/main/java/org/apache/accumulo/manager/ManagerTime.java

```diff
@@ -64,6 +64,8 @@ public ManagerTime(Manager manager, AccumuloConfiguration conf) throws IOExcepti
       throw new IOException("Error updating manager time", ex);
     }
 
+    // Previous versions of this class did not use a shared thread pool, it created its own
+    // Timer instance.
```

Comment:
I'm not sure what purpose this comment serves. It's documenting a previous version of the code. It doesn't apply to the code here. If it is needed and kept, it should be reworded to describe the current code and the reason for the current implementation, not reference a previous version that no longer applies.

Reply:
Comment removed in e71efba